### PR TITLE
product-card was created

### PR DESCRIPTION
**Pull request by yaroslavna-horban**
- [DEMO LINK](https://yaroslavna-horban.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://yaroslavna-horban.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,52 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img 
+        class="product-card__image" 
+        src="images/imac.jpeg" 
+        alt="Apple A1419 iMac 27$quot; Retina 5K Monoblock">
+
```
Comment (Anton-Zhytonbaiev) [inline src/index.html:18]:
```suggestion
      <img 
        class="product-card__image" 
        src="images/imac.jpeg" 
        alt="Apple A1419 iMac 27$quot; Retina 5K Monoblock"
      >

```

File: src/styles/blocks/product-card.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.product-card__title {
+  width: 166px;
+  font-size: 12px;
+  margin: 0 16px 4px;
```
Comment (Anton-Zhytonbaiev) [inline src/styles/blocks/product-card.css:18]: also cheklist point 7 (maybe i wrong)

**Review by Anton-Zhytonbaiev — CHANGES_REQUESTED** [review]
Good job (i don't find smile, just imagine something good)

File: src/styles/blocks/product-card.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-width: max-content;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
```
[the remark this answers is not in the document]
Comment (yaroslavna-horban) [reply src/styles/blocks/product-card.css:12]: I may be wrong but it isn't a block

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

**Pull request closed without merging**